extensions/wikia/Recirculation/services/CurationCMSService.class.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+class CurationCMSService {
+	static protected $instance = null;
+	private $slug;
+	private $limit;
+
+	const API_BASE = 'https://services.wikia.com/curation-cms/stories/feed/slug/';
```

Comment:
if curation-cms is pandora service then we should use swagger to generate service client

Reply:
Let's do it after AB test.